addons/binding/org.openhab.binding.mihome/ESH-INF/thing/sensor_wleak_aq1.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="sensor_wleak.aq1">
```

Comment:
nearly right ;-)
please make it _aq1 otherwise OH wont map the thing to this config...

Reply:
Oops! Done ;-)